CMakeLists.txt

```diff
@@ -16,7 +16,40 @@ catkin_package(
   )
 
 add_library(${PROJECT_NAME} src/realtime_clock.cpp)
-target_link_libraries(${PROJECT_NAME} ${catkin_LIBRARIES} ${Boost_LIBRARIES})
+target_link_libraries(${PROJECT_NAME} PUBLIC ${catkin_LIBRARIES} ${Boost_LIBRARIES})
+
+# Unit Tests
+if (CATKIN_ENABLE_TESTING)
+  catkin_add_gtest(realtime_box_tests test/realtime_box_tests.cpp)
```

Comment:
How do you feel about using GMOCK instead? The majority wouldn't change now but I think that moving forward it'd be valuable.

Reply:
Switched to gmock in ac9bf90f27c2c8464c9c7cf2775960ca1ba3d95f

Reply:
Great!